### DISCUSSION TITLE
Docker + Kepler.gl

### DISCUSSION THREAD

**Discussion by devpoppy** [keplergl/kepler.gl#2836] [Q&A] [answered]
Hi Kelper team
I tried to deploy the kepler demo-app with docker but got several issues
Could you please provide the dockerfile for this?

**Comment by ibgreen** [comment]
This is not planned by current maintainers. Converting to a discussion in case there is community input.

**Reply by devpoppy** [reply]
yeah, okay

**Reply by delfrrr** [reply]
I'm maintaining an open-source backend for Kepler called Dekart: it’s designed to be deployed as a single Docker container.
It includes optional Postgres support and is much easier to get running than the demo app.

Happy to share setup details if you're interested:

https://github.com/dekart-xyz/dekart

**Reply by jo-chemla** [reply]
Useful discussion here mentions using `export NODE_OPTIONS=--openssl-legacy-provider`, although we did not take the time to build the keplergl container image

https://github.com/keplergl/kepler.gl/issues/2024

**Comment by igorDykhta** [comment, marked as answer]
```
cp .env.template .env   # set MapboxAccessToken
docker compose -f docker/docker-compose.yml up kepler-prod

```
Then open http://localhost:8080/. Details: [docker/README.md](https://github.com/keplergl/kepler.gl/blob/master/docker/README.md).
